api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/BaseReferenceHandlingTranslator.java

```diff
@@ -133,11 +132,6 @@ protected Reference createPractitionerReference(@Nonnull Provider provider) {
 				sb.append(person.getPersonName().getFullName());
 			}
 			
-			if (provider.getIdentifier() != null) {
-				reference.setIdentifier(new Identifier().setValue(provider.getIdentifier()));
-				sb.append(" (").append(FhirConstants.IDENTIFIER).append(": ").append(provider.getIdentifier()).append(")");
-			}
```

Comment:
Why did we remove this whole block? The ticket was just to get rid of the text display. The reference identifier should still be set.

Reply:
Yeah you are right seems like I overlooked the `setIdentifier` statement and removed the whole block.
Have fixed it now. Thanks for pointing this out.